### PR TITLE
Update Cloth.js

### PR DESCRIPTION
**Pull request by mghoffmann**
I fixed a typo in a function name ("satisfy" instead of "satisify") and removed an assignment that seems unnecessary- I can't find any references to the value of the assigned variable after the assignment.

### CODE REVIEW

**Comment by WestLangley** [comment]
>and removed an assignment that seems unnecessary

Did you test this change?

File: examples/js/Cloth.js [hunk 1]
```diff
@@ -90,7 +90,6 @@ Particle.prototype.integrate = function( timesq ) {
 	newPos.multiplyScalar( DRAG ).add( this.position );
 	newPos.add( this.a.multiplyScalar( timesq ) );
 
-	this.tmp = this.previous;
```
Comment (Mugen87) [inline examples/js/Cloth.js]: If you undo this change, we can merge the PR and at least fix the typo.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by mrdoob** [comment]
I'll merge and clean up if needed.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Haha!

screen shot 2017-04-19 at 14 23 33

**Comment by WestLangley** [comment]
Clearly -- a PR not tested.

**Comment by mrdoob** [comment]
Fixed.

**Comment by mghoffmann** [comment]
My bad. I searched the code for all references of this.tmp, and it seemed
like it was never used. I guess it was. A more meaningful name might be
good?

Is there some set procedure for testing code changes for this project, so I
don't mess it up again? I hope to contribute what little I can to this.

**Comment by mrdoob** [comment]
> Is there some set procedure for testing code changes for this project, so I
don't mess it up again?

Loading the example in the browser is a good start 😉

**Pull request merged**